index.html

```diff
@@ -13,28 +13,149 @@
+    <li>html</li>
+    <li>css</li>
+    <li>js</li>
+   </ul>
+   <div class="see-project">
+    <p><a href="#">See project</a></p>
+   </div>
+  </div>
+
+  <div class="project">
+   <img src="./assest/img/Snapshoot Portfolio (1).png" alt="">
+   <h2>Multi-Post
+    Stories</h2>
+   <ul class="toni">
+    <h5>CANOPY</h5>
+    <li> <span></span> Back End Dev</li>
+    <li> <span></span> 2022</li>
+   </ul>
+   <p class="decription">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="language">
+     <li>html</li>
+     <li>css</li>
+     <li>js</li>
+    </ul>
+    <div class="see-project">
+     <p><a href="#">See project</a></p>
+    </div>
+   </div>
+
+
+   <div class="project">
+    <img src="./assest/img/Snapshoot Portfolio (2).png" alt="">
+    <h2>Tonic</h2>
+    <ul class="toni">
+     <h5>CANOPY</h5>
+     <li> <span></span> Back End Dev</li>
+     <li> <span></span> 2022</li>
+    </ul>
+    <p class="decription">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     <ul class="language">
+      <li>html</li>
+      <li>css</li>
+      <li>js</li>
+     </ul>
+     <div class="see-project">
+      <p><a href="#">See project</a></p>
+     </div>
+    </div>
+
+    <div class="project">
+     <img src="./assest/img/Snapshoot Portfolio (3).png" alt="">
+     <h2>Tonic</h2>
+     <ul class="toni">
+      <h5>CANOPY</h5>
+      <li> <span></span> Back End Dev</li>
+      <li> <span></span> 2022</li>
+     </ul>
+     <p class="decription">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="language">
+       <li>html</li>
+       <li>css</li>
+       <li>js</li>
+      </ul>
+      <div class="see-project">
+       <p><a href="#">See project</a></p>
+      </div>
+     </div>
+ </section>
+ <section class="about">
+  <div class="content">
+  <h2>About Myself</h2>
+  <p class="description">
+   Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+  </p>
+  <h5>LET'S CONNECT</h5>
+  <ul class="links">
+   <li><a title="linkdin" href="https://www.linkedin.com/in/naji-ali-55b5421b3/"><i class="fa-brands fa-linkedin fa-2x"></i></a></li>
+   <li><a title="github" href="https://github.com/naji-97"><img src="./assest/icons/Vector.svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (3).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (4).svg" alt="" srcset=""></a></li>
+   <li><a title="twiter" href="https://twitter.com/Naji_2023"><img src="./assest/icons/Vector (2).svg" alt="" srcset=""></a></li>
+  </ul>
+  <div class="see-resume">
+   <p><a href="#">See resume</a></p>
+  </div>
```

Comment:
As I commented before, kindly add the various states for this element As optional, I suggest you use a button tag to make styling it faster.